### ISSUE TITLE
Get access to learned coefficients of nested model

### ISSUE THREAD

**Issue by tlienart** [alan-turing-institute/MLJ.jl#334]
I'm not sure how that's supposed to happen. Let's say you have this:

```julia
@pipeline HotReg(hot = OneHotEncoder(),
                 reg = LassoRegressor())
hotreg = HotReg()
r  = range(model, :(reg.lambda), lower=1e-2, upper=1e9, scale=:log10)
tm = TunedModel(model=model, ranges=r, tuning=Grid(resolution=50),
                measure=rms)
mtm = machine(tm, X, y)
fit!(mtm)
```

I can get back the lambda without  issue, but getting the coefficients of the regression with that lambda is a bit painful to get, I had to do:

```julia
mtm.fitresult.fitresult.machine.fitresult
```

is there  a nicer way that I would be unaware of? if not, should we add one?

**Comment by ablaom** [comment]
Okay, I've substituted LassoRegressor with RidgeRegressor (from MultivariateStats) because of this issue: https://github.com/alan-turing-institute/MLJModels.jl/issues/133

Yes, there is a missing interface point here. The named  tuple `fitted_params(mtm)` should include, in addition to `best_model`,  an item `best_fitted_params`. This should have value `fitted_params(mach)` where `mach=mtm.fitresult.`  If this were implemented, your next step would be to call `fitted_params` on `fitted_params(mach)`, which would give you:

```julia
julia> fitted_params(ans)
(machines = Any[NodalMachine{RidgeRegressor} @ 8…48, NodalMachine{OneHotEncoder} @ 1…29],
 fitted_params = NamedTuple[(coefficients = [-0.0961981, 0.0551593, -0.0418801, -0.0400429, 0.915533, 0.0223552, -0.8198, 0.330313, -0.0169058, -0.76756, 0.00835806, -0.754541], intercept = 43.0458), (fitresult = OneHotEncoderResult @ 7…95,)],)
```

(Currently you must do `fitted_params(mtm.fitresult)` to get this). This named tuple has two keys: `machines` (all machines in the underlying learning network) and `fitted_params` (the result of calling `fitted_params` on each machine in turn). So your coefficients are then 

```julia
fitted_parms(mtm).best_fitted_params.fitted_params[1].coefficients
```
